topi/python/topi/vision/yolo2/region.py

```diff
@@ -0,0 +1,64 @@
+"""
+REGION Operator
+====================
+**Author**: `Siju Samuel <https://github.com/siju-samuel/>`_
```

Comment:
let us not put authors as they are already represented per git commit history, unless there is special reason to do so

Reply:
Done